### PR TITLE
removes quotes from config command when returning single setting value

### PR DESCRIPTION
**Pull request by naringas**
# Pull Request Check List

This is just a reminder about the most common mistakes. Please make sure that you tick all *appropriate* boxes.  But please read our [contribution guide](https://poetry.eustace.io/docs/contributing/) at least once, it will save you unnecessary review cycles!

- [x] Added **tests** for changed code.
- [x] Updated **documentation** for changed code.

based on develop, see issue #820

### CODE REVIEW

File: poetry/console/commands/config.py [hunk 1]
```diff
@@ -130,7 +130,11 @@ def handle(self):
                 )
 
                 for value in values:
-                    self.line(value[1])
+                    if isinstance(value[1], str) and value[1][0] == value[1][-1] == '"':
```
Comment (drunkwcodes) [inline poetry/console/commands/config.py]: Would you like to add ws check in here?

Comment (naringas) [reply poetry/console/commands/config.py]: sorry, what do you mean by "ws"?

Comment (drunkwcodes) [reply poetry/console/commands/config.py]: White space characters.

Comment (naringas) [reply poetry/console/commands/config.py]: I cannot get any command such as `cd $(poetry config settings.virtualenvs.path)` to work no matter what I do.

if `poetry config settings.virtualenvs.path` returns the strings with whitepsace escaped, e.g. a setting with the vaule `/some path/dir with space` returns `/some\ path/dir\ with\ space` usring the command inside like `ls $(...)` still doesn't work.

So I don't know how to proceed. I guess I will drop the whole thing.

**Comment by naringas** [comment]
no good

**Pull request closed without merging**